Makefile

```diff
@@ -43,10 +43,12 @@ $(TOOLS_BIN_DIR):
 	mkdir -p $(TOOLS_BIN_DIR)
 
 # Image URL to use all building/pushing image targets
+BOOTSTRAP_IMG_TAG ?= v0.2.0
 BOOTSTRAP_IMG ?= ghcr.io/cluster-api-provider-k3s/cluster-api-k3s/bootstrap-controller:v0.2.0
 
 # Image URL to use all building/pushing image targets
-CONTROLPLANE_IMG ?= ghcr.io/cluster-api-provider-k3s/cluster-api-k3s/controlplane-controller:v0.2.0
+CONTROLPLANE_IMG_TAG ?= v0.2.0
```

Comment:
hmm is there value in having a hard coded tag here?

Reply:
Hmm, Idk maybe not but if it is not set to some default it'll fail.

Reply:
ok ... we can look to improve this in the future.